### PR TITLE
changing the cwd to the download dir in _download_call() this should …

### PR DESCRIPTION
**Pull request by jessicapilling**
…solve issue #282

### CODE REVIEW

File: daxa/mission/xmm.py [hunk 1]
```diff
@@ -260,6 +260,11 @@ def _download_call(observation_id: str, insts: List[str], level: str, filename:
         :return: A None value.
         :rtype: Any
         """
+        # The astroquery module will only download to the cwd, so need to change to where we want the data downloaded
```
Comment (jessicapilling) [inline daxa/mission/xmm.py:263]: this is my rudimentary solution to this problem. I wonder if we should also put in an issue to the astroquery github, I cant believe this is an intended behaviour? either way I have tested this and it gets rid of the error

File: daxa/mission/xmm.py [hunk 2]
```diff
@@ -260,6 +260,11 @@ def _download_call(observation_id: str, insts: List[str], level: str, filename:
...
+        og_dir = os.getcwd()  # storing this info so we can return here after the download is done
+        dest_dir_path = filename.replace(observation_id, '')  # this is a static method, so I cant use self.raw_data_path
+        os.chdir(dest_dir_path)  # changing to where we want the file downloaded
```
Comment (jessicapilling) [inline daxa/mission/xmm.py:266]: this is my rudimentary solution to this problem. I wonder if we should also put in an issue to the astroquery github, I cant believe this is an intended behaviour? either way I have tested this and it gets rid of the error

File: daxa/mission/xmm.py (hunk 1 above)
Comment (DavidT3) [inline daxa/mission/xmm.py:263]: Not going to require the changes, but I do not like comments on the end of lines personally - they can easily end up getting in the way/going over the maximum number of characters in a line when you change the code.

File: daxa/mission/xmm.py [hunk 3]
```diff
@@ -276,7 +281,8 @@ def _download_call(observation_id: str, insts: List[str], level: str, filename:
                 # Download the requested data
                 AQXMMNewton.download_data(observation_id=observation_id, level=level, filename=filename)
             except Exception as err:
-                raise Exception("{oi} data failed to download.").with_traceback(err.__traceback__)
```
Comment (DavidT3) [inline daxa/mission/xmm.py:279]: Oooh did I forget to format it with the ObsID when I wrote that??

**Review by DavidT3 — APPROVED** [review]

**Pull request merged**